src/fat.rs

```diff
@@ -26,7 +26,7 @@ pub fn create_fat_filesystem(
         .truncate(true)
         .open(out_fat_path)
         .unwrap();
-    let fat_size_padded_and_rounded = ((needed_size + 1024 * 64 - 1) / MB + 1) * MB;
+    let fat_size_padded_and_rounded = (((needed_size + 1024 * 64 - 1) / MB + 1) * MB) + MB;
```

Comment:
To me it's more readable without the extra parentheses:

```suggestion
    let fat_size_padded_and_rounded = ((needed_size + 1024 * 64 - 1) / MB + 1) * MB + MB;
```

Reply:
Had to use a hard reset and force-push to incorporate both of these changes without also inadvertently turning 1 commit into 2, but done.